src/app/app.component.ts

```diff
@@ -34,6 +37,9 @@ export class MyApp {
       // Here you can do any higher level native things you might need.
       this.statusBar.styleDefault();
       this.splashScreen.hide();
+      if (window.cordova) {
+        window.open = cordova.InAppBrowser.open;
```

Comment:
This is no longer needed. See: https://github.com/keycloak/keycloak/pull/4514

Reply:
@wtrocki is this change in a released version of the Keycloak JS adapter? We tried using the 'cordova' adapter on init earlier and it didn't use the in-app browser, only the system one.

Reply:
https://github.com/keycloak/keycloak-js-bower/blob/master/dist/keycloak.js#L1173-L1176

Reply:
I actually confirmed the same problem on the Android. This was just luck/time sensitive bug that was happening because we did not wait for cordova device ready script and things were crashing randomly. Since IOS platform has quite different plugin <-> webview init times than Android this was almost always failing.